airbyte-integrations/connectors/source-trustpilot/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
```

Comment:
Oh, great, so poetry _is_ setup — thank you. Then make sure that readme does not mention `MAIN_DEPENDENCIES` and such

Reply:
Done!